src/main/java/com/zendesk/maxwell/schema/SchemaPosition.java

```diff
@@ -85,7 +87,7 @@ private void store(BinlogPosition newPosition) {
 		if ( newPosition == null )
 			return;
 
-		String sql = "INSERT INTO `maxwell`.`positions` set "
+		String sql = "INSERT INTO `" + this.schemaDatabaseName + "`.`positions` set "
```

Comment:
looks like I could specify a default database in the jdbc connection URI.

the other option would be to use connection.setCatalog to specify the db.